index.js

```diff
@@ -4,32 +4,236 @@ const logo = document.querySelector('#logo');
+
+        language: {
+          li1: 'HTML',
+          li2: 'Css',
+          li3: 'JavaScript',
+          li4: 'bootstrap',
+          li5: 'query',
+          li6: 'react.js',
+        },
+      },
+    ];
+
+    projects.forEach((value, index) => {
+      // console.log(value.key , index)
+      if (value.key === cardid) {
+        doc.innerHTML = `<div class="popup">
+        <button id="pop-close">&times;</button>
+        <div class="">
+          <h2 class="title1"> ${projects[index].heading}</h2>
+          <ul class="frame1" id="pop-frame">
+            <li class="A Li">${projects[index].frame.li1}</li>
+            <li class="BC Li">
+              <i class="bi bi-circle-fill icon"></i> ${projects[index].frame.li2}
+            </li>
+            <li class="BC Li"><i class="bi bi-circle-fill icon"></i> ${projects[index].frame.li3}</li>
+          </ul>
+          <img
+            class="card-img"
+            id="pop-img"
+            src=${projects[index].img}
+            alt=${projects[index].alt}
+          />
+        </div>
+        <div class="pop-box1">
+          <p class="pop-para">
+          ${projects[index].para1}
+          <br>
+          <br>
+          ${projects[index].para1}
+          </p>
+          <div class="small-box">
+            <ul class="box2-ul pop-ul">
+              <li class="box2-li pop-li">${projects[index].language.li1}</li>
+              <li class="box2-li pop-li">${projects[index].language.li2}</li>
+              <li class="box2-li pop-li">${projects[index].language.li3}</li>
+              </ul>
+              <ul class="box2-ul pop-ul">
+              <li class="box2-li pop-li">${projects[index].language.li4}</li>
+              <li class="box2-li pop-li">${projects[index].language.li5}</li>
+              <li class="box2-li pop-li">${projects[index].language.li6}</li>
+            </ul>
+            <div class="pop-buttons">
+              <button class="big-btn btn_btn pop-btn">See Live  <i class="bi bi-arrow-up-right-circle-fill"></i> </button>
+              <button class="big-btn btn_btn pop-btn">See source  <i class="bi bi-github"></i> </button>
+            </div>
+          </div>
+        </div>
+        </div>`;
+      }
+    });
+
+    // console.log(projects[0].id)
+    // console.log(projects[0].key)
+    // console.log(projects[0].heading)
+    // console.log(projects[0].frame.li1)
+    // console.log(projects[0].frame.li2)
+    // console.log(projects[0].frame.li3)
+    // console.log(projects[0].img)
+    // console.log(projects[0].alt)
+    // console.log(projects[0].para)
+    // console.log(projects[0].language.li1)
+    // console.log(projects[0].language.li2)
+    // console.log(projects[0].language.li3)
+    // console.log(projects[0].language.li4)
+    // console.log(projects[0].language.li5)
+    // console.log(projects[0].language.li6)
```

Comment:
- [ ] Kindly consider removing the commented code as it makes our codebase look messy.